frontend/packages/dev-console/src/components/taskruns/events/__tests__/TaskRunDetailsPage.spec.tsx

```diff
@@ -36,7 +36,7 @@ describe('TaskRunDetailsPage:', () => {
   it('Should contain three tabs in the Details Page', () => {
     const wrapper = shallow(<TaskRunDetailsPage {...taskRunDetailsPageProps} />);
     const { pages } = wrapper.props();
-    expect(pages).toHaveLength(3);
+    expect(pages).toHaveLength(4);
```

Comment:
Point in case to a bad test. Did we worry about the number of pages dynamically changing on us? You just added a new tab, and then went an updated a test, likely after it failed on you and not as a proactive measure to drive a TDD scenario... so, what value does this test get us?

Reply:
This was added as part of some separate PR and I felt simply to update it. I've removed the block. @andrewballantyne

Reply:
I understand this was written in another PR, but it was an example to help you understand what happens when you write a "static verification" test. There isn't really any gain here, nothing is protected from accidental breaks, and you just end up modifying the test as you change the content.